.openshift-ci/slack/notify.sh

```diff
@@ -1,8 +1,10 @@
 #!/usr/bin/env bash
 set -eo pipefail
 
-# shellcheck disable=SC1091
-source .openshift-ci/env.sh
+shopt -s nullglob
+for cred in /tmp/secret/**/[A-Z]*; do
```

Comment:
not one to fix now, perhaps, but we have this pattern in most (all?) scripts but we also have a function in `.openshift-ci/drivers/scripts/lib.sh` that does it too. At some point soon we really need to consolidate all the commonly used patterns

Reply:
Yep, that's what I thought as well (and originally the issue even was that I assumed there *was* a global env.sh). Will address it in a follow-up PR.